packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -218,8 +263,6 @@ export async function handleConfirmation(args: {
         const eventTypeSlug = updatedBookings[index].eventType?.slug || "";
 
         const isFirstBooking = index === 0;
-        const videoCallUrl =
```

Comment:
The original booking didn't have videoCallUrl in metadata, that's why it was missing in the workflow emails that trigger when a meeting was confirmed